run/test/specs/message_checks_groups_android.spec.ts

```diff
@@ -0,0 +1,546 @@
+import { androidIt } from "../../types/sessionIt";
+import { sleepFor } from "./utils";
+import { newUser } from "./utils/create_account";
+import { createGroup } from "./utils/create_group";
+import {
+  SupportedPlatformsType,
+  closeApp,
+  openAppThreeDevices,
+} from "./utils/open_app";
+import { runScriptAndLog } from "./utils/utilities";
+
+async function sendImageGroup(platform: SupportedPlatformsType) {
+  const testGroupName = "Message checks for groups";
+  const { device1, device2, device3 } = await openAppThreeDevices(platform);
+  // Create users A, B and C
+  const [userA, userB, userC] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+    newUser(device3, "Charlie", platform),
+  ]);
```

Comment:
the more you wan it to add the fixtures the harder to will be

Reply:
What do you mean? @Bilb